recipes/gpi_core/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set version = "2.0.0rc" %}
+{% set build_num = "0" %}
+{% set sha256 = "8dc97c93e48f57febc67773107276dbbdadb8c8effa75bea18a475c07266442a" %}
+
+package:
+  name: gpi_core
+  version: {{ version }}
+
+source:
+  url: https://github.com/gpilab/core-nodes/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  folder: repo_contents
+
+build:
+  number: {{ build_num }}
+  detect_binary_files_with_prefix: True
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [linux]
+    - {{ compiler('cxx') }}  # [linux]
+    - gcc  # [mac]
+  host:
+    - cytoolz
+    - eigen =3
+    - fftw
+    - gpi >=1.1.4
+    - numpy >=1.11
+    - python {{ python }}
+  run:
+    - {{ pin_compatible('eigen') }}
+    - {{ pin_compatible('fftw') }}
+    - {{ pin_compatible('gpi') }}
+    - matplotlib >=2.1,<3
+#    - matplotlib <2  # [linux]
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('python') }}
+    - scipy
+
+test:
+  imports:
+    - gpi
+    - gpi_core
+  source_files:
+    - repo_contents/testNets/*
+
+about:
+  home: http://gpilab.com
+  license: GPL3
```

It's nice to use the SPDX identifier for consistency:
```suggestion
  license: GPL-3.0
```